src/App.css

```diff
@@ -1206,4 +1213,84 @@
     border-top: 1px solid var(--border-color);
     padding-top: 24px;
   }
-} 
\ No newline at end of file
+} 
+
+/* Priority Styles */
+.task-priority-selector {
+  margin-bottom: 1rem;
+  padding: 12px;
+  background: var(--hover-bg);
+  border-radius: 6px;
+}
+
+.task-priority-selector label {
+  display: block;
+  margin-bottom: 8px;
+  font-size: 12px;
+  text-transform: uppercase;
+  color: var(--text-secondary);
+  font-weight: 600;
+  letter-spacing: 0.5px;
+}
+
+.priority-options {
+  display: flex;
+  gap: 8px;
+}
+
+.priority-button {
+  padding: 6px 12px;
+  border-radius: 4px;
+  cursor: pointer;
+  transition: all 0.2s ease;
+  font-weight: 500;
+  font-size: 12px;
+  border: 2px solid transparent;
+  flex: 1;
+  text-align: center;
+}
```

Comment:
![kody code-review](https://img.shields.io/badge/kody-code--review-brightgreen)

```javascript
.priority-button {
  padding: 6px 12px;
  border-radius: 4px;
  cursor: pointer;
  transition: transform 0.2s ease;
  font-weight: 500;
  font-size: 12px;
  border: 2px solid transparent;
  flex: 1;
  text-align: center;
}
```
The transition property on .priority-button should specify which properties to animate for better performance, rather than using 'all'.



<sub>Talk to Kody by mentioning @kody</sub>

<sub>Was this suggestion helpful? React with 👍 or 👎 to help Kody learn from this interaction.</sub><!-- kody-codereview -->&#8203;
&#8203;